packages/server/src/routes/grantsSavedSearch.js

```diff
@@ -68,7 +68,7 @@ router.delete('/:searchId', requireUser, async (req, res) => {
     }
 
     if (deleteSuccess) {
-        res.status(200).send('OK');
+        res.status(200).json({ status: 'OK' });
```

Comment:
Make into JSON so the client handles it. I could also update the client to not try to parse it, but this was quick & it seemed good to make it consistent with the other endpoints anyway.